lxc/console_unix.go

```diff
@@ -0,0 +1,41 @@
+// +build !windows
+
+package main
+
+import (
+	"io"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/lxc/lxd/shared/logger"
+)
+
+func (c *consoleCmd) getStdout() io.WriteCloser {
+	return os.Stdout
+}
+
+func (c *consoleCmd) controlSocketHandler(control *websocket.Conn) {
```

Comment:
Shouldn't be needed for console

Reply:
That's the same implementation that we had since 2.0.0 and it makes since because we need to exclude windows builds

Reply:
Indeed, we do need this code to handle sigwinch, not sure why I left that comment, must have gotten confused with the other signal handling code which doesn't make sense for console.

Reply:
Perfectly fine. This was an RFC anyway so thanks for the detailed review!